### ISSUE TITLE
Highlight.js error

### ISSUE THREAD

**Issue by marcustisater** [postcss/postcss.org#230]
From PR #223 

```/Users/halvves/dev/opensource/postcss.org/node_modules/highlight.js/styles/tomorrow.css:4
.hljs-comment {
^
SyntaxError: Unexpected token .
    at Object.exports.runInThisContext (vm.js:76:16)
    at Module._compile (module.js:528:28)
    at Module._extensions..js (module.js:565:10)
    at Object.require.extensions.(anonymous function) [as .js] (/Users/halvves/dev/opensource/postcss.org/node_modules/babel-register/lib/node.js:152:7)
    at Module.load (module.js:473:32)
    at tryModuleLoad (module.js:432:12)
    at Function.Module._load (module.js:424:3)
    at Module.require (module.js:483:17)
    at require (internal/module.js:20:19)
    at eval (eval at <anonymous> (/Users/halvves/dev/opensource/postcss.org/node_modules/.cache/phenomic/phenomic.node.bundle.js:5:29610), <anonymous>:1:18) 
```

ping @halvves @MoOx if someone would want to take a look at it.

**Comment by MoOx** [comment]
Looks like a classic webpack error when a file is not caught by any loader. Be sure that one of your css loader will catch this file https://github.com/postcss/postcss.org/blob/9eda9a49b086e151a9f8627d0e6129e62bd37120/webpack.config.babel.js#L61-L83

**Comment by meuwka** [comment]
@MoOx @marcustisater Looks like phenomic issue https://github.com/MoOx/phenomic/issues/639
fixed in 0.16.1 https://github.com/MoOx/phenomic/releases/tag/0.16.1

**Comment by marcustisater** [comment]
Sweet, thanks for the info. PR welcome otherwise I will try get it fix this weekend.
